.github/workflows/main.yml

```diff
@@ -9,19 +9,19 @@ on:
   schedule:
     - cron: '0 0 * * *' # Every day at midnight
 
+defaults:
+  run:
+    shell: bash
```

Comment:
Interesting. Is it running zsh on macOS?

Reply:
Or PowerShell on Windows? 😄

Reply:
Powershell on windows by default was the issue.

Reply:
Holy cow. I was just kidding around. 😄

Reply:
heh, holy cow is usual state of play for DevOps =)